docs/user-guide/src/quick-start.md

```diff
@@ -107,7 +107,7 @@ virt-install \
   --name bmh-vm-01 \
   --description "Virtualized BareMetalHost" \
   --osinfo=ubuntu-lts-latest \
-  --ram=4096 \
+  --ram=4096 `# use 8192 for Scenario 2` \
```

Comment:
```suggestion
  --ram=4096  \
```
Put the comment on L105?

Reply:
See d0d975cd9ee125f261f963332e870f2844119f63